src/components/moldes/AddMoldesForm.js

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+
+export default function AddMoldesForm(props) {
+  const isEditing = props.editing;
+  return (
+    <React.Fragment>  
+        <Grid item xs={12} >
+          {isEditing ? <TextField
+            disabled
+            id="id"
+            name="id"
+            label="ID del molde"
+            variant="standard"
+            defaultValue={props.mock.id}
+          /> : <TextField
+            required
+            id="id"
+            name="id"
+            label="ID del molde"
+            variant="standard"
+          />}
+        </Grid>
+        <Grid item xs={12} >
+          {isEditing ? 
+            <TextField
+              required
+              id="nombreMolde"
+              name="nombreMolde"
+              label="Nombre del molde"
+              variant="standard"
+              defaultValue={props.mock.nombreMolde}
```

Comment:
no se puede checar lo del is editing aqui mismo? tipo
defaultValue={ isEditing  ? props.mock.nombreMolde : ""}